### DISCUSSION TITLE
crossing=unmarked is causing a severe confusion (some actions taken, effects to be reviewed furing future testing)

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3341] [Feedback]
People are highly confused by crossing quests on crossings  not marked by zebra or other signage. Many notes are being left.

It applies to crossing type, islands, tactile paving quests.

It is also clearly visible while reviewing StreetComplete notes being left.

I have no good idea what to do with it right now.

Ignore problem?

Add note explaining that it is an unmarked crossing if quest is being solved for crossing=unmarked?

Tweak interface somehow? Adding one more answer in crossing type quest that will also mark crossing=unmarked?

Skip crossing=unmarked in this questions?

Try to invent a new tagging?

https://www.openstreetmap.org/note/2875822 + https://www.openstreetmap.org/note/2875826 are typical

In my sample so far out of 6 people who encountered crossing quest every single person was really confused by it getting asked for places such as https://wiki.openstreetmap.org/wiki/File:51_Edgewater_Boulevard_(36709048234).jpg https://wiki.openstreetmap.org/wiki/File:Crossing_without_painting_on_road_with_lowered_kerbs.png

**Comment by kmpoppe** [comment]
We also had a lively discussion about that in @t/OSMde, without a clear result.

While reading the toxic discussion in the forum, essentially accusing @westnordost of inventing tagging and forcing it through with the help of SC wasn't a very good starting point of finding a solution as well.

My idea would be to deactivate the crossing quest by default and giving a message when the user wants to activate it, something like "please read the wiki what and why SC thinks is a crossing" or something.

JMTC

Kai

**Reply by westnordost** [reply]
The discussion in the forum you refer to was about the `AddCrossing` quest type I believe. The examples shown by @matkoniecz however are about the `AddCrossingType` quest though. The latter is only shown if there is already a `highway=crossing` marked.

**Comment by timothywashere** [comment]
I admit, I also find it confusing, especially when there is no crossing at all.

I haven't read all the Issues for this quest, nor the OSM Wiki on crossings.

If there is no crossing but footpaths meet a road I select "High Kerbs". If it the road crossed the footpath I would have to leave a note saying it's not a crossing.

It would be easier to understand if the first question was along the lines of "Is there a crossing here?"
Or if the first question gave options for whether there was a marked foot crossing the road, unmarked foot crossing the road or if the road crosses the footpath, what we in the UK call a "Dropped Kerb".

**Comment by westnordost** [comment]
> https://www.openstreetmap.org/note/2875822 + https://www.openstreetmap.org/note/2875826 are typical

This is about the `AddCrossingType` quest though. The node already has `highway=crossing` tagged, so this is beyond the question whether such a situation should be tagged with `highway=crossing` at all.

I'm reading from this issue https://github.com/openstreetmap/iD/issues/8463 that iD suggests to its users to set (i.e. "fix") `highway=crossing` on any intersection between a road-way and a footway, including driveways. 
So as long as this is the default behavior, we can expect to have many of such situations. And of course also beyond that, if this ever gets changed.

One solution would be to not ask crossing-related quests on nodes that are at are at the intersection to a `highway=service` even if `highway=crossing` is tagged. Or even if it is `highway=footway + footway=sidewalk` (i.e. not `footway=crossing`).

**Reply by matkoniecz** [reply]
> One solution would be to not ask crossing-related quests on nodes that are at are at the intersection to a highway=service

Maybe.

Right now I think that SC should stop asking about crossing detail (tactile paving, island) for ones tagged with `crossing=unmarked`. Not because it is useless to collect such data, but because such questions are sadly often really confusing.

If that would be rejected I would propose to show extra guidance for user - below question.

> Or even if it is highway=footway + footway=sidewalk (i.e. not footway=crossing).

This would not help too much at least in Poland where [such places](https://wiki.openstreetmap.org/wiki/File:Crossing_without_painting_on_road_with_lowered_kerbs.png) are common, confusing and can have `footway=crossing` or be without `footway`

**Reply by westnordost** [reply]
> Right now I think that SC should stop asking about crossing detail (tactile paving, island) for ones tagged with crossing=unmarked. 

But the crossings in your example are not marked with `crossing=unmarked`.

**Reply by matkoniecz** [reply]
I run into similar issues (especially with crossing island quest) for https://www.openstreetmap.org/node/311337269 - https://www.google.pl/maps/@50.0645698,19.946532,3a,54.9y,70.91h,91.2t/data=!3m6!1e1!3m4!1sVmnWcYe2UMTPNwmOrs5n6g!2e0!7i13312!8i6656
![screen02](https://user-images.githubusercontent.com/899988/136074763-0955ac15-86a0-4768-ae56-e70721288d62.png)


And https://www.openstreetmap.org/node/4022441217 https://www.google.pl/maps/@50.0635342,19.9465338,3a,75y,85.86h,80.43t/data=!3m6!1e1!3m4!1sHznbzBEjB69dVsIeLwBYWw!2e0!7i13312!8i6656

![screen03](https://user-images.githubusercontent.com/899988/136074915-9970ac64-06e7-4db7-930d-43cb1af7530a.png)

where this would help

And crossings looking like https://www.google.pl/maps/@50.0688187,19.9527682,3a,75y,117.95h,66.46t/data=!3m9!1e1!3m7!1s2cOKBpx46RGJGe4n8MV7oA!2e0!7i13312!8i6656!9m2!1b1!2i35 and https://www.google.pl/maps/@50.0674224,19.9315299,3a,35.7y,158.32h,87.6t/data=!3m9!1e1!3m7!1sAbgmaMpkfmxscSD1X6nA1g!2e0!7i13312!8i6656!9m2!1b1!2i35 etc

**Reply by westnordost** [reply]
Maany such "crossings" actually do have a lowered kerb, so since the answer will not always the same, it makes sense to ask.

**Reply by westnordost** [reply]
Whoa, what is this
https://www.openstreetmap.org/way/296334617#map=19/50.06372/19.94618

Btw looks like a really nice neighbourhood

**Reply by smichel17** [reply]
> One solution would be to not ask crossing-related quests on nodes that are at are at the intersection to a `highway=service` even if `highway=crossing` is tagged

I think that's essentially the same idea as https://github.com/streetcomplete/StreetComplete/pull/3301 / https://github.com/streetcomplete/StreetComplete/issues/3294, applied more widely

**Reply by matkoniecz** [reply]
> Whoa, what is this
> https://www.openstreetmap.org/way/296334617#map=19/50.06372/19.94618

import made by fan of nanomapping

**Reply by westnordost** [reply]
specifically #3294, yes

**Reply by andrewharvey** [reply]
I'd still ask it for highway=service, just maybe not for highway=service + service=driveway.

**Reply by westnordost** [reply]
Most `highway=service` have no subtag and StreetComplete does also not ask its users to specify a subtag. So, many driveways do not actually have the driveway-tag set. Also, I'd say the same applies for parking aisles and drive-throughs (some). Most service ways that enter from the public road to a private property go over the sidewalk, I'd say.

Or well, maybe depends on the place. At an IKEA, probably not, at a (parking lot to a) grocery store, probably yes.

**Reply by andrewharvey** [reply]
Alternatively opt in service=alley when tagged but exclude other service=* or no service tag. Alleys are usually streets just like residential, with curbs, sidewalks, crossings.

**Comment by andrewharvey** [comment]
I don't get the confusion, crossing is just a place where you cross the road, and sometimes the crossing is unmarked, but it's still a crossing. The photo is clear there are no markings. I'm loving the crossing type quest, it's great for improving pedestrian mapping in OSM.

**Reply by matkoniecz** [reply]
People look at such location and declare that there is no pedestrian crossing there, despite considering it as a place to cross as a pedestrian.

I observed it while testing in Poland but from looking at notes it seems widespread in general :(

**Reply by mnalis** [reply]
It might also be that the mapper is having different [understanding](https://www.thefreedictionary.com/crossing) of what is meant by the "crossing" exactly.

For example, when two ways make an `X`-shape, that would be understood as ***crossing*** since _"one way **crosses over** the other way"_. However it the two ways are instead *connected* in `T`-shape or `Y`-shape, it might be understood as **junction**, not as a **crossing**.

Now add to the mix the fact that the [crosswalk](https://en.wikipedia.org/wiki/Crosswalk) is almost never positioned exactly at the **crossing** (ie. place where the two roads cross), but almost always moved a little to the side of one road (and often repeated so we usually get `4` crosswalks on `1` crossing), and I can see where much of the confusion could come from.

**Comment by smichel17** [comment]
Maybe this could be helped with translation updates?

In the original post here, you link two notes by the same user. They read:

> not a crossing. this is an ramp in a sidewalk for cars to enter the parking lot

> not a crosswalk

It seems to me that this person is using "crossing" and "crosswalk" interchangeably; I *think* the latter is defined as a marked crossing.

**Reply by mnalis** [reply]
That might well be the case. It also doesn't help that the answer `crossing without road markings` has the text all over the most important part of the picture. Perhaps replace by `No marked crosswalk`, and positioned on the upper (not relevant) part of the picture, (or different picture if that is easier) maybe?

**Reply by matkoniecz** [reply]
Repositioning text is unfeasible (would be an ugly and confusing hack), but new photo is possible. 

"no marked crosswalk" is not ideal, as there are sometimes some weird unusual marking that also qualify as markings

**Reply by smichel17** [reply]
What if we removed "crossing" from all of the translations? After all, the question already says it's asking about a crossing.
That would leave us with:

- Controlled by traffic lights
- Marked
- Without road markings (better phrasing: "No road markings" or "Not marked on road")

Hopefully that would focus people on the important words in the sentence; the user in the original example might be comfortable answering "no markings" instead of getting fixated on the word "crossing". Also it would save an additional line to not block the picture.

**Reply by matkoniecz** [reply]
![screen](https://user-images.githubusercontent.com/899988/136663902-93eb7021-f036-4324-930c-e4424631183a.png)

**Reply by smichel17** [reply]
Did you change the margins, too? On my phone it's 3 lines:

![image](https://user-images.githubusercontent.com/6563664/136664067-3841282c-db02-4349-88c1-61cd6f0a991f.png)

**Reply by matkoniecz** [reply]
Likely your screen is smaller than what I emulated.

I emulate 1080 x 2340, 440 dpi screen - matching Pixel 5.

**Reply by matkoniecz** [reply]
Weird. I tried on lower size screen with lower density, but....

![screen01](https://user-images.githubusercontent.com/899988/136664965-88d5959b-048b-4992-a1d8-17c938ee5008.png)

**Reply by westnordost** [reply]
Regarding the "new possible photo" - it looks like a really broad sidewalk. Is this a street for cars?

**Reply by westnordost** [reply]
(Rhetoric question). In a replacement picture, it should at least be obvious that it is a street.

Idea with removing "crossing" from the wording is good!

**Reply by matkoniecz** [reply]
> Idea with removing "crossing" from the wording is good!

I like the idea but I am unsure about replacements...

"What kind of passage is this?"

"What kind of pedestrian passage is this?"

(I am not liking either)

> In a replacement picture, it should at least be obvious that it is a street.

OK, I will try to capture something again.

**Reply by westnordost** [reply]
From the wording of the answers, not the wording of the question!

**Reply by smichel17** [reply]
```diff
 What kind of crossing is this?
-- Crossing controlled by traffic lights
-- Marked crossing
-- Crossing without road markings
+- Controlled by traffic lights
+- Marked
+- No road markings
```

**Reply by matkoniecz** [reply]
https://www.geograph.org.uk/photo/6501361

https://www.geograph.org.uk/photo/3062086

are sort of fitting, but without humans walking across

**Reply by matkoniecz** [reply]
@smichel17 Do you want to make PR or should I steal that?

**Reply by smichel17** [reply]
Steal away, I have too many open PRs already :)

**Reply by hungerburg** [reply]
As someone, who considers crossing=unmarked overused, the Broadway, Horsforth one https://www.geograph.org.uk/photo/6501361 to me belongs to the set, where crossing=unmarked is a must map.

**Reply by matkoniecz** [reply]
![screen](https://user-images.githubusercontent.com/899988/138165005-a9445e30-2d85-4f97-84e3-b8202ac9bec8.png)


I found a location for a photo, though I need a better day for photo taking (more clouds to reduce difference between lit/unlit parts) or visit during different hour.

Some debris in the background but should be good enough.

**Reply by matkoniecz** [reply]
And new one, I am quite happy about it
![screen04](https://user-images.githubusercontent.com/899988/138287337-4e128f5b-d930-48d0-8f55-1f38288d3b78.png)

**Reply by hungerburg** [reply]
Looks a bit like the woman would not trust the floor to carry her weight. At least, she walks, as if she went on thin ice. Perhaps, it might be possible, to make it look like, _this crossing is unmarked, the floor should safe, but attention has to be a bit more on oncoming traffic, as on a marked one_?

**Reply by smichel17** [reply]
I'd be interested to see it from a different angle, closer to the [photo linked above](https://www.geograph.org.uk/photo/6501361)

![photo from above](https://s0.geograph.org.uk/geophotos/06/50/13/6501361_3e1a3a4c_original.jpg)

But maybe from a slightly different angle, as if you are standing a meter or two to the right of the photographer.

**Reply by smichel17** [reply]
I can probably also find a crossing to take similar photos, myself… I know I'm making many photo requests, here :)

**Reply by hungerburg** [reply]
Reading up on the subject, https://www.fhwa.dot.gov/publications/research/safety/04100/04100.pdf perhaps the woman in the picture of Matesusz' above, actually does best for her safety by NOT looking at oncoming traffic. Divers are likely to interpret such as yielding. Perhaps, for illustrative purposes only, it might be nice though, if the person on the picture showed the palms of their hand directed at the photographer standing on the carriageway, as an improvised stop sign, so to say.

**Reply by matkoniecz** [reply]
Extra note: due to oneway road layout she was actually looking in direction where incoming vehicle (cyclist) was present (and keeping eye contact was also mentioned somewhere as a good idea).

On topic of making more photos: I ma not really planning as I am happy with this one and getting it took a ridiculous amount of time.

**Comment by smichel17** [comment]
> new photo is possible

Normally the way I'd identify such an unmarked crossing *as a crossing* would be the place it touches the road (e.g. a lowered curb that indicates I should cross here). But with the current photo, that's not really visible.

![current photo](https://raw.githubusercontent.com/streetcomplete/StreetComplete/db4a862d04602e3d0b2c5234519f7f6376a9f017/app/src/main/res/drawable-xxhdpi/crossing_type_unmarked.jpg)

I think an ideal photo would be from the perspective of the sidewalk, as if I am about to cross. That way the other side of the street would be clearly visible above the text, and my side of the street would be large enough that it should be somewhat-visible, even under the text.

**Reply by smichel17** [reply]
Note that this is more clear in the full-size photo linked above, which I think is of the same place, and shows the lowered curb on the left.

![the referenced photo](https://upload.wikimedia.org/wikipedia/commons/2/2c/Crossing_without_painting_on_road_with_lowered_kerbs.png)

@matkoniecz It looks like you took these photos. Would it be convenient to visit this crossing some time and take some other photos from different angles? If not, I could try to find an unmarked crossing in my area which would be suitable.

It might be best if whoever takes the photos brings a friend (or helpful bystander) to stand in the crossing as if they are walking, as an extra hint that it is still a crossing.

**Reply by matkoniecz** [reply]
See https://github.com/streetcomplete/StreetComplete/discussions/3341#discussioncomment-1452809 for a new possible photo

**Reply by matkoniecz** [reply]
> If not, I could try to find an unmarked crossing in my area which would be suitable.

If you manage to take photo then it would be great! I will try again tomorrow, hopefully weather will be still allowing this.

> It might be best if whoever takes the photos brings a friend (or helpful bystander) to stand in the crossing as if they are walking, as an extra hint that it is still a crossing.

I also did this - friend is better as it (1) easier to ask them to repeatedly move across crossing, as you take photos (2) easier to make clear how this photo will be used (distributed, but person is going to not be recognizable)

**Reply by mnalis** [reply]
Do try to take different angles while you are there @matkoniecz; maybe a 90 degrees rotated (first person view) might be better than this view (0 degrees), or something in between (30 or 45 degrees?). And/or try to take it with wider angle, too. Problem with this picture for example is that lowered kerb on both sides is barely visible at the edges, so a lot a context is missing. I agree that person(s) crossing it are a must to establish what we're showing exactly...

**Reply by timothywashere** [reply]
I think any photo will be hard to understand as they are shrunk down onto a small screen. A cartoon type image would probably be clearer and easier to understand

**Comment by hungerburg** [comment]
Users answering the AddCrossingType quest are adding valuable information to openstreetmap. Crossing=unmarked is a valid answer, as there are valid mappings for this.

I understand the confusion though: The Edgewater picture does not show a crossing at all, simply because, it does not even show a street. As such are expected to become much more numerous due to the policy of the most widely used editor, placing notes is not going go scale. I suggest to create a new "crossing=fake" tag.

PS: @matkoniecz - regarding another picture of yours: A crossing marked with a traffic sign only is a marked crossing, isn't it?

**Reply by matkoniecz** [reply]
> A crossing marked with a traffic sign only is a marked crossing, isn't it?

¯\\\_(ツ)_/¯

https://wiki.openstreetmap.org/wiki/Tag:crossing%3Dmarked has "just road markings" but I suspect that different people do it differently. I would use unmarked.

If I would really care about this kind of data I would invent tagging scheme including separately specifying to tag specifically vertical traffic signs and road marking.

Or at least document it at https://wiki.openstreetmap.org/wiki/Tag:crossing%3Dmarked

> I suggest to create a new "crossing=fake" tag. 

At this moment I have a backlog of proposals and I admit that I hope that someone else will spend time on that. Either way it would be nice to have `highway=pedestrian_passage` `crossing=driveway_passage` `crossing=fake` or something.

But I want first to finish some of already started proposals.

**Reply by hungerburg** [reply]
I guess I will have to write a pull request for the iD editor to bring the point with the fake crossings to their attention.

Meanwhile I just argue here, that the crossing with the sign is legally just as binding as the crossing marked with paintings on the road. At least in Germany and Austria, traffic signs and road markings are functionally equivalent. I see the value of collecting the information in the information, that the crossing is a "protected" one, where pedestrians get the preference. This is the information, that e.g.. routers for accessibility need. With unmarked, this is not necessarily so.

That is why I always add crossing_ref=zebra to features, that have a zebra; because marked is indeterminate on the kind of marking.

PS: The "deprecation" of crossing=zebra by the iD editor perhaps came from, that in the US crossings rarely are marked with zebra but mostly just lines to the left/right of the "footway=crossing" - so the SC picture for marked, showing a zebra, might create confusion there?

**Reply by matkoniecz** [reply]
> the crossing with the sign is legally just as binding as the crossing marked with paintings on the road. At least in Germany and Austria, traffic signs and road markings are functionally equivalent. I see the value of collecting the information in the information, that the crossing is a "protected" one, where pedestrians get the preference. This is the information, that e.g.. routers for accessibility need. With unmarked, this is not necessarily so.

Note that would requite inventing a new tagging scheme.

> so the SC picture for marked, showing a zebra, might create confusion there?

If someone has a good image (square, with space at the bottom for label, without debris, openly license) it would be possible to have a different image for USA.

But that would require either taking such photo (and I never visited USA) or finding something at https://commons.wikimedia.org/

**Comment by hungerburg** [comment]
The distinction, marked | unmarked, is following a US terminology; Myself, from Europe, have some problems. Can someone here help me: What kind are the ones in the pictures below?

- https://upload.wikimedia.org/wikipedia/commons/a/af/Curb_extensions_at_crosswalk.jpg
- https://upload.wikimedia.org/wikipedia/commons/a/a2/Seattle_-_Man_in_crosswalk%2C_1946_%2837639658721%29.jpg
- https://upload.wikimedia.org/wikipedia/commons/1/1d/Crosswalks_in_disrepair_%2847798792291%29.jpg

In my understanding, they would be unmarked… regardless the paint, as it is not following the typical pattern.

**Reply by riQQ** [reply]
As I understand it, the pattern doesn't matter. Any markings on the carriageway for a crossing count as marked. Your examples are all marked / uncontrolled in my opinion.

**Reply by andrewharvey** [reply]
- marked as in road markings or signage so that cars can see their is a crossing there
- unmarked as no signage or road markings warning cars there is a crossing

**Reply by westnordost** [reply]
Yeah, all the shown pictures show marked crossings.

**Reply by hungerburg** [reply]
Thank you all. Indeed, in many places this **is** the typical pattern, the picture from Canada cannot be any more explicit on that. My motivation to ask such a silly question: Trying to get at a symbol, to represent unmarked crossings on a map.

**Comment by BalooUriza** [comment]
On Wed, Nov 3, 2021 at 6:36 AM hungerburg ***@***.***> wrote:

> Thank you all. Indeed, in many places this *is* the typical pattern, the
> picture from Canada cannot be any more explicit on that. My motivation to
> ask such a silly question: Trying to get at a symbol, to represent unmarked
> crossings on a map.
>

JOSM's rendering of highway=crossing, crossing=unmarked is "good enough",
in that I know what it's trying to tell me.

**Comment by BalooUriza** [comment]
Do you need one to be taken?  I have a kajillion crosswalks handy if
necessary.

**Reply by mnalis** [reply]
I agree with @matkoniecz that https://github.com/streetcomplete/StreetComplete/discussions/3341#discussioncomment-1514424 is quite nice, have enough context of pavment on left and right sides, visible perpendicular street at the top that connects to current one coming from below, and have enough space on bottom half of the screen (so displaying text there won't make picture unintelligible). So it looks very good to me :+1: ; certainly much better than current one. 

If someone is really bored and thinks they can make even better shot, well, it's their free time; but I think the suggested one is quite good.

**Reply by matkoniecz** [reply]
Image linked by @mnalis is already used (maybe in not-yet-released version). 

Like with all other images: if anyone will find or make a better one it would be welcomed! For example this one still has some unwanted visual debris.

But replacement for this one would need to have

- no road markings at all
- something like lowered curb
- no more debris than this one
- human walking across crossing

and other things mentioned by @mnalis 

While this is not ideal, it still took quite a long time to take this photo (I tried several other locations earlier, looking for suitable place with a good background took quite  along time).

tl;dr This image can be improved, if better would be available it would be used - but it will take noticeable effort to make clearly better one.
